docs/docs-content/clusters/public-cloud/aws/eks.md

```diff
@@ -39,118 +47,146 @@ The following tags should be added to the virtual private network (VPC) public s
 
 ## Deploy an AWS Cluster
 
-Use the following steps to provision a new AWS EKS cluster:
+Use the following steps to deploy an AWS cluster in which to provision an EKS cluster.
 
-1. Ensure you are in the correct project scope.
+1. Log in to [Palette](https://console.spectrocloud.com/).
 
+2. Ensure you are in the correct project scope.
 
-2. Navigate to the left **Main Menu** and click on **Clusters**
+3. From the left **Main Menu** select **Clusters**, and click on the **Add New Cluster** button.
 
+4. Select **Deploy New Cluster** on the next page Palette displays. This will allow you to deploy a cluster using your own cloud account. 
 
-3. Click on **Add New Cluster**
+5. Select **AWS** and click on the **Start AWS Configuration** button.
 
+6. Fill out the following basic information, and click **Next** to continue.
 
-4. You will receive a prompt asking you if you want to deploy a new cluster or import an existing cluster. Click on **Deploy New Cluster**
+  | **Field** | **Description** |
+  |-----------|-----------------|
+  | **Cluster Name**| A custom name for the cluster. |
+  | **Description**| Use the description to provide context about the cluster.|
+  | **Tags**| Assign any desired cluster tags. Tags on a cluster are propagated to the Virtual Machines (VMs) deployed to the target environments. Example: `region:us-east-1a` or `zone:vpc-private-us-east-1a`.|
+  | **Cloud Account** | If you already added your AWS account in Palette, select it from the **drop-down Menu**. Otherwise, click on **Add New Account** and add your AWS account information. |
 
+  To learn how to add an AWS account, review the [Add an AWS Account to Palette](add-aws-accounts.md) guide.
 
-5. Select **AWS** and click on **Start AWS Configuration**
 
+### Create an EKS Cluster
 
-6. Populate the wizard page with the following information: name, description, tags and AWS account. Tags on a cluster are propagated to the VMs deployed to the target environments. Click on **Next** after you have filled out all the required information.
+Use the following steps to provision a new EKS cluster.
 
-7. Selected **Managed Kubernetes** and click on your cluster profile that supports AWS EKS. Click on **Next**.
+1. Select **EKS** as the **Managed Kubernetes**. 

```

Comment:
this sounds clunky?

Reply:
I agree - I debated how to guide the user in the UI. Changed it to "Select **EKS** listed under **Managed Kubernetes**.